test/integration/services/data/test-remove-user-by-username.js

```diff
@@ -0,0 +1,28 @@
+const expect = require('chai').expect
+
+const removeUserByUsername = require('../../../../app/services/data/remove-user-by-username')
+const userRoleHelper = require('../../../helpers/data/user-role-helper')
+
+var username
+var insertedData = []
+
+describe('/services/data/remove-user-by-username', function () {
+  before(function () {
+    return userRoleHelper.addUsers()
+      .then(function (ids) {
+        username = ids[0].username
+        insertedData = ids
+      })
+  })
+
+  it('should remove user role by username', function () {
+    return removeUserByUsername(username)
+      .then(function (result) {
+        expect(result).to.be.greaterThan(0)
```

Comment:
I don't think its necessary since the remove will tell you the number of records removed, which should be more than 0. However, I have added what was suggested to extend the test for further coverage.

Reply:
Coll